src/dune_lang/package_dependency.ml

```diff
@@ -63,4 +96,4 @@ let to_dyn { name; constraint_ } =
 let equal { name; constraint_ } t =
   Package_name.equal name t.name
   && Option.equal Package_constraint.equal constraint_ t.constraint_
-;;
+;;
```

Comment:
Try to avoid adding/removing the trailing newline from the diff (ie. only include the meaningful part of your change). Some editors automatically mess with file endings. The problem with checking this in is that the next person who touches the file will have to deal with the changed line ending from _their_ editor and so on. My advice would be to configure your editor to leave file endings alone, and also to use `git add -p` or similar to only stage the meaningful changes.